database/102-up-timescale/20231025144021-spooler_add-amm-lp-rewards.sql

```diff
@@ -0,0 +1,29 @@
+-- migrate:up
+
+CREATE TABLE amm_positions (
+	tick_lower INT NOT NULL,
+	tick_upper INT NOT NULL,
+	position_id uint256 UNIQUE NOT NULL,
+
+	-- token id
```

Comment:
Specify these comments to refer to the token address versus the token name (and also the context!)